tests/e2e/multi_svc_util.go

```diff
@@ -0,0 +1,285 @@
+	framework.Logf("Remove an ESXi host from cluster command : %s", removeHostFromCluster)
+	_, err := exec.Command("/bin/sh", "-c", removeHostFromCluster).Output()
+	if err != nil {
+		framework.Logf("Error: %v\n", err)
+		return false, fmt.Errorf("couldn't execute command: %s, error: %s",
+			removeHostFromCluster, err)
+	}
+	return true, nil
+}
+
+// moveHostToCluster method is used to move a host to cluster
+func moveHostToCluster(clusterPath string, hostIP string) error {
+	moveHostToCluster := govcLoginCmd() + "govc cluster.mv -cluster " + clusterPath + " " + hostIP
+	framework.Logf("Move a host to cluster command : %s", moveHostToCluster)
+	_, err := exec.Command("/bin/sh", "-c", moveHostToCluster).Output()
+	if err != nil {
+		framework.Logf("Error: %v\n", err)
+		return fmt.Errorf("couldn't execute command: %s, error: %s",
+			moveHostToCluster, err)
+	}
+	return nil
+}
+
+// getVcSessionIDforSupervisor method returns list of vc session id for a supervisor id and returns error if any
+func getVcSessionIDforSupervisor(supervisorId string) ([]string, error) {
+	getSessionId := govcLoginCmd() + "govc session.ls | grep '" + supervisorId + "' | awk '{print $1}'"
+	framework.Logf("Get Vc session ID for cluster command : %s", getSessionId)
+	result, err := exec.Command("/bin/sh", "-c", getSessionId).Output()
+	if err != nil {
+		return []string{}, fmt.Errorf("couldn't execute command: %s, error: %s",
+			getSessionId, err)
+	}
+	sessionIds := strings.Split(strings.TrimSpace(string(result)), "\n")
+	return sessionIds, nil
+}
+
+// killVcSessionIDs remove vc session id for a supervisor cluster
+func killVcSessionIDs(sessionIds []string) error {
+	var govcLogin string = govcLoginCmd()
+	for _, sessionId := range sessionIds {
+		removeSessionIdCmd := govcLogin + "govc session.rm " + sessionId
+		framework.Logf("Remove vc session id from cluster - command : %s", removeSessionIdCmd)
+		_, err := exec.Command("/bin/sh", "-c", removeSessionIdCmd).Output()
+		if err != nil {
+			framework.Logf("Error: %v\n", err)
+			return fmt.Errorf("couldn't execute command: %s, error: %s",
+				removeSessionIdCmd, err)
+		}
+	}
+	return nil
+}
+
+// getSvcConfigSecretData returns data obtained fom csi config secret
+// in namespace where CSI is deployed
+func getSvcConfigSecretData(client clientset.Interface, ctx context.Context,
+	csiNamespace string) (e2eTestConfig, error) {
+	var vsphereCfg e2eTestConfig
+	currentSecret, err := client.CoreV1().Secrets(csiNamespace).Get(ctx, configSecret, metav1.GetOptions{})
+	if err != nil {
+		return vsphereCfg, err
+	}
+	originalConf := string(currentSecret.Data[vsphereCloudProviderConfiguration])
+	vsphereCfg, err = readConfigFromSecretString(originalConf)
+	if err != nil {
+		return vsphereCfg, err
+	}
+
+	return vsphereCfg, nil
+}
+
+// getSvcDatastoreNames method is used to fetch datastore details from a multi-supervisor testbed
+func getSvcDatastoreNames(sshClientConfig *ssh.ClientConfig,
```

Comment:
use an apt name
also the clusters can be in different DCs

Reply:
changed it to getDatastoreNamesFromDCs